frontend/public/keycloak.json

```diff
@@ -1,6 +1,6 @@
 {
   "realm": "pims",
-  "auth-server-url": "http://keycloak:8080/auth",
+  "auth-server-url": "http://localhost:8080/auth",
```

Comment:
Latest keycloak configuration no longer requires the same domain name for api and app